src/arkit/ARKitDevice.js

```diff
@@ -181,17 +194,17 @@ export default class ARKitDevice extends PolyfilledXRDevice {
 			}
 
 			switch(type){
-				case 'head-model':
+				case 'viewer':
 					enqueueOrExec(function () { 
 						resolve(that._headModelMatrix) 
 					})
 					return
-				case 'eye-level':
+				case 'local':
 					enqueueOrExec(function () { 
 						resolve(that._eyeLevelMatrix) 
 					})
 					return
-				case 'stage':
+				case 'stage': // @TODO: Update
```

Comment:
I haven't touched `stage` key yet because it doesn't seem to be used from our polyfill and examples.

Reply:
stage was for VR contexts in WebVR, I don't think we're talking about having it in WebXR are we?

Reply:
I meant, no `stage` key in the latest spec.

https://immersive-web.github.io/webxr/#xrreferencespace-interface

So we need to replace the key, or remove the key if we don't need. But I haven't touched the `stage` key yet to save my time because `stage` key doesn't seem to be used from the polyfill or the examples. So I just left the TODO comment so far.

Reply:
So, we will need to support local, local-floor (with the floor likely changing over time as we get a better guess based on the planes we see), and unbounded.  

I'm not sure I understand the different between local and unbounded.

Reply:
You meant, we don't need viewer?

Reply:
no, no.  viewer we definitely need, sorry about that!  I was thinking about the other ones when I typed that.

Reply:
OK OK. I see, no problem.

The TODO comment seemed unclear. So I removed `stage` key and added `local-floor`, `unbound`, and `bounded-floor` but they return no supported error for now. We may update to support them if needed later.

Reply:
We'll definitely want to support at least unbounded.  It's the natural coordinate space for something like ARKit, as it describes exactly how ARKit works (you can move an unlimited distance, and the origin of the space changes to be near you).

Also, we probably need a TODO to pay attention to IF the local origin changes (which it can do) and to issue reset messages on all the coordinate system aside from the unbounded ones.

Reply:
Can we do them in another PR? I personally want to focus on letting the existing features work on the latest WebXR API and want to add new features after that.

Reply:
absolutely, I literally meant "put a TODO in the comments." :)

Reply:
I want to confirm my understanding. Does this comment look good to you? Let me know if I misunderstood anything.

```
// @TODO: In unbounded reference space the native origin can change
//        if the user moves long distance from their starting point.
//        If that happens we may need to issue reset messages
//        on all the coordinate system.
```

Reply:
It's the opposite. 
```
// @TODO: support unbounded reference space.
//        in ARKit, the native origin can change as the user moves around. 
//        in unbounded, the space origin can change.  But in other space
//        such as local and local-floor, if the origin changes, a reset event should
//        be triggered on the coordinate system (per the spec).
```

Reply:
Thanks. I was missing reset event description in the spec.

Reply:
Added TODO comments, thanks.